newIDE/app/src/AssetStore/index.js

```diff
@@ -54,6 +57,27 @@ export type AssetStoreInterface = {|
   onClose: () => void,
 |};
 
+const identifyAssetPackKind = ({
```

Comment:
I know PublicAssetPack should be the same as PublicAssetPack, but as it's not, should we say `isPrivate === !!assetPack.content` ?
(this props is only present on private packs)

Reply:
This feels convoluted to search in the arrays.
And in the future, if both types merge to have the same props, we could add a `type: private | public` and use it?

Reply:
I wonder if it's too fragile, and the day we update the backend to send a content for public packs, we now have wrong analytics. So we broke analytics by adding a field on the backend?

Reply:
I'll go ahead with this and this is embedded in a function, so we an always simplify this later.